### ISSUE TITLE
Why do I lose Names assigned by zero-shot after applying outlier reduction?

### ISSUE THREAD

**Issue by mzhadigerov** [MaartenGr/BERTopic#2041]
I am using zero-shot topic list to assign topics to clusters of documents:

```python
topic_model = BERTopic(
    embedding_model=embedding_model,
    min_topic_size=50,
    zeroshot_topic_list=zeroshot_topic_list,
    zeroshot_min_similarity=.40,
    representation_model=representation_model,
    vectorizer_model=vectorizer_model,
)

topics, _ = topic_model.fit_transform(documents=docs, embeddings=embeddings)
```
It is working as expected. I can see my custom labels under `Name` column, when I do `topic_model.get_topic_info()`

However, there are too many outliers and I wanted to reduce outliers:

```python
new_topics = topic_model.reduce_outliers(docs, topics, strategy="embeddings", embeddings=embeddings)
topic_model.update_topics(docs, topics=new_topics)
```

However, after I do `topic_model.get_topic_info()` again, my zero-shot labels are gone. Why?
![image](https://github.com/MaartenGr/BERTopic/assets/55549813/e184da1c-22f4-4b79-b29b-8181c59aebcf)
![image](https://github.com/MaartenGr/BERTopic/assets/55549813/0de8ca2a-129e-4cee-b6b2-26fd5cffff12)

**Comment by MaartenGr** [comment]
@mzhadigerov Most likely, you are losing the names since using `.update_topics` creates new topic representations. Instead, I would advise assigning your selected topic representations to [custom labels](https://maartengr.github.io/BERTopic/getting_started/topicrepresentation/topicrepresentation.html#custom-labels) **before** running `.update_topics`. That way, you will always save your custom labels that you can use throughout your analysis and in visualizations.
